From 957d3f9f65d47fd046d9a282e4de8a9a5b694032 Mon Sep 17 00:00:00 2001
From: Gustavo Padovan <gustavo.padovan@collabora.co.uk>
Date: Tue, 31 May 2016 16:59:06 -0300
Subject: [PATCH 54/88] staging/android: clean up #includes in the sync
 framework
Git-commit: d79892ad0b97ec23645cffa5406edb4eb18cb553
Patch-mainline: v4.8-rc1
References: fate#321223

Most of the includes there are not necessary anymore.

Signed-off-by: Gustavo Padovan <gustavo.padovan@collabora.co.uk>
Reviewed-by: Daniel Vetter <daniel.vetter@ffwll.ch>
Reviewed-by: Sumit Semwal <sumit.semwal@linaro.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Patrik Jakobsson <pjakobsson@suse.de>
---
 drivers/staging/android/sync.c       |  6 ------
 drivers/staging/android/sync.h       |  3 ---
 drivers/staging/android/sync_debug.c | 16 ----------------
 3 files changed, 25 deletions(-)

diff --git a/drivers/staging/android/sync.c b/drivers/staging/android/sync.c
index aab80ec7bce6..bb12d86ccb1d 100644
--- a/drivers/staging/android/sync.c
+++ b/drivers/staging/android/sync.c
@@ -14,14 +14,8 @@
  *
  */
 
-#include <linux/debugfs.h>
 #include <linux/export.h>
-#include <linux/kernel.h>
-#include <linux/sched.h>
-#include <linux/seq_file.h>
 #include <linux/slab.h>
-#include <linux/uaccess.h>
-#include <linux/anon_inodes.h>
 
 #include "sync.h"
 
diff --git a/drivers/staging/android/sync.h b/drivers/staging/android/sync.h
index 02ecf440d306..54c515bd9a1a 100644
--- a/drivers/staging/android/sync.h
+++ b/drivers/staging/android/sync.h
@@ -13,9 +13,6 @@
 #ifndef _LINUX_SYNC_H
 #define _LINUX_SYNC_H
 
-#include <linux/types.h>
-#include <linux/kref.h>
-#include <linux/ktime.h>
 #include <linux/list.h>
 #include <linux/spinlock.h>
 #include <linux/fence.h>
diff --git a/drivers/staging/android/sync_debug.c b/drivers/staging/android/sync_debug.c
index 2733cc34ecf8..864ad0103435 100644
--- a/drivers/staging/android/sync_debug.c
+++ b/drivers/staging/android/sync_debug.c
@@ -15,22 +15,6 @@
  */
 
 #include <linux/debugfs.h>
-#include <linux/export.h>
-#include <linux/file.h>
-#include <linux/fs.h>
-#include <linux/kernel.h>
-#include <linux/poll.h>
-#include <linux/sched.h>
-#include <linux/seq_file.h>
-#include <linux/slab.h>
-#include <linux/uaccess.h>
-#include <linux/anon_inodes.h>
-#include <linux/time64.h>
-#include <linux/sync_file.h>
-#include <linux/types.h>
-#include <linux/kconfig.h>
-
-#include "uapi/sw_sync.h"
 #include "sync.h"
 
 #ifdef CONFIG_DEBUG_FS
-- 
2.12.0
